source/patterns/@aws-solutions-constructs/aws-alb-lambda/lib/index.ts

```diff
@@ -146,7 +146,7 @@ export class AlbToLambda extends Construct {
     this.lambdaFunction = defaults.buildLambdaFunction(this, {
       existingLambdaObj: props.existingLambdaObj,
       lambdaFunctionProps: props.lambdaFunctionProps,
-      vpc: this.vpc,
```

Comment:
we do this on purpose, as it prevents typo errors in maintenance if we add members later. As it is the only edit of this sort and we are far from consistent I'll ignore it.